atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/table/common/TableTasks.java

```diff
@@ -163,6 +164,7 @@ private static PartialCopyStats copyInternal(
         return stats;
     }
 
+    @AllowedRangeRequest(justification = "???")
```

Comment:
size estimator is ok because..? we only ever read one batch?

Reply:
Today at least: only used by the large internal product, which is not generally deployed with Cassandra.

(This probably should itself be restricted, though probably something for another day...)